packages/now-cli/src/util/errors-ts.ts

```diff
@@ -1236,3 +1236,26 @@ export class BuildError extends NowError<'BUILD_ERROR', {}> {
     });
   }
 }
+
+/**
+ * This Error subclass can be used by the `output.prettyError()` function.
+ */
+export class LinkableError extends Error {
```

Comment:
I think you can use `import { NowBuildError } from '@vercel/build-utils'` instead of creating a new class

Reply:
Ok good call.